README.md

```diff
@@ -2,27 +2,28 @@ Simit is a new programming language designed to make it easy to write
 high-performance code to compute on sparse systems.  For more information see
 [simit-lang.org](http://simit-lang.org).
 
-Building Simit
-==============
-To build Simit you must install CMake 2.8.3 or later and LLVM 3.4.2.  See
-http://www.cmake.org/cmake/resources/software.html for a copy of CMake. If your
-system does not have packages for LLVM you can download it at
-http://llvm.org/releases/download.html. You must then make sure llvm-config is
-available in your path.
+Build Simit
+===========
+To build Simit you must install
+[CMake 2.8.3 or greater](http//www.cmake.org/cmake/resources/software.html) and
+[LLVM 3.7 or greater](http://llvm.org/releases/download.html).
+You must then make sure llvm-config is available in your path.
 
 If you want to build LLVM yourself you can check it out using SVN:
 
-    svn co https://llvm.org/svn/llvm-project/llvm/branches/release_34/ llvm-3.4.2
+    svn co https://llvm.org/svn/llvm-project/llvm/branches/release_37 llvm3.7
 
-Then build it:
+then build it:
 
-    cd llvm-3.4.2
-    ./configure --disable-terminfo --enable-optimized --enable-assertions --enable-targets=x86
+    cd llvm3.7
+    mkdir build
+    cd build
+    cmake -DLLVM_ENABLE_TERMINFO=OFF -DLLVM_TARGETS_TO_BUILD="X86;NVPTX" -DLLVM_ENABLE_ASSERTIONS=ON -DCMAKE_BUILD_TYPE=Release ..
```

Comment:
Should we add the "-DLLVM_ENABLE_RTTI=ON" flag here until we figure out a way around that?